### PR TITLE
Fix Escape in hypothetical

### PR DESCRIPTION
**Pull request by timotree3**

### CODE REVIEW

**Comment by timotree3** [comment]
done testing

File: client/src/game/ui/keyboard.ts [hunk 1]
```diff
@@ -105,7 +111,6 @@ function keydown(event: JQuery.KeyDownEvent) {
 
     // Escape = If in an in-game replay, exit back to the game
     replay.exit();
-    return;
```
Comment (Zamiell) [inline client/src/game/ui/keyboard.ts:108]: should this be removed?

Comment (timotree3) [reply client/src/game/ui/keyboard.ts]: good point

**Comment by Zamiell** [comment]
thanks

**Pull request merged**